libcontainerd/client_windows.go

```diff
@@ -126,7 +126,12 @@ func (clnt *client) Create(containerID string, checkpoint string, checkpointDir
 		mds[i] = hcsshim.MappedDir{
 			HostPath:      mount.Source,
 			ContainerPath: mount.Destination,
-			ReadOnly:      mount.Readonly}
+			ReadOnly:      false}
```

Comment:
@stevvooe Turns out if you omit the last `,`, a `}` on the same line is syntactically correct for the compiler and for godoc. However, if you put the `}` on a new line, the compiler requires the trailing comma. Learn something every day :innocent: 

Anyway, put the brace on the new line instead and pushed an update.